setup.py

```diff
@@ -41,6 +41,7 @@
     packages=find_packages(exclude=['tests*', 'benchmarks*']),
     include_package_data=True,
     install_requires=[
+        'avro',
```

Comment:
the project does not list avro as dependency but instead uses [yelp-avro==1.9.2](https://pypi.python.org/pypi/yelp_avro/1.9.2), which is yelp's version of avro(this version has extra functionality which are absent in apache/avro).